README.md

```diff
@@ -238,8 +238,8 @@ will give the same result as
     -q --target world
 
 It is this property which leads us to an Applicative interface
-instead of a Monadic one, as all option must be considered in
-parallel, and can not depend on the output of other options.
+instead of a Monadic one, as all options must be considered in
+parallel, and cannot depend on the output of other options.
```

Comment:
controversy on cannot!

Reply:
I think cannot is more an American thing, I don't think I've ever used it.

Thank you for pointing out the option(s) error. Much appreciated.

Reply:
https://en.oxforddictionaries.com/usage/cannot-or-can-not